lib/fizzy/execute.cpp

```diff
@@ -414,6 +419,71 @@ T fnearest(T value) noexcept
         return t;
 }
 
+
+union Num32
+{
+    unsigned int i32;
+    float f32;
+};
+
+union Num64
+{
+    uint64_t i64;
+    double d64;
+};
+
+float fabs(float input) noexcept
+{
+    Num32 result;
+    result.f32 = input;
+    result.i32 &= ABS_MASK;
+    return result.f32;
+}
+
+double dabs(double input) noexcept
+{
+    Num64 result;
+    result.d64 = input;
+    result.i64 &= ABS_MASK64;
+    return result.d64;
+}
+
+float fneg(float input) noexcept
+{
+    Num32 result;
+    result.f32 = input;
+    result.i32 ^= SIGN_MASK;
+    return result.f32;
+}
+
+double dneg(double input) noexcept
+{
+    Num64 result;
+    result.d64 = input;
+    result.i64 ^= SIGN_MASK64;
+    return result.d64;
+}
+
+float copy_sign(float f1, float f2) noexcept
```

Comment:
Please use `a` and `b` as parameter names, similar to what the other binary ops are doing.